leaper.py

```diff
@@ -343,10 +358,11 @@ def check_one_request(self, req):
             return False
 
         request_ok = ReviewBot.ReviewBot.check_one_request(self, req)
-        has_correct_maintainer = self.maintbot.check_one_request(req)
+        if not self.ibs:
+            has_correct_maintainer = self.maintbot.check_one_request(req)
+            self.logger.debug("has_correct_maintainer: %s", has_correct_maintainer)
```

Comment:
ideally this needs to be reimplemented too. For SLE bugownership in one of the layered projects is used, rather than maintainership in devel projects.